<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JUQ15IP" failures="1" tests="3" name="Test" time="9.71" errors="0" timestamp="2020-11-18T12:49:43 EST">
  <testcase classname="com.vndly.testCases.TC_SubmitForm" name="TC_fill_Student_RegistrationForm" time="0.553"/>
  <testcase classname="com.vndly.testCases.TC_SubmitForm" name="validateText" time="0.046"/>
  <testcase classname="com.vndly.testCases.TC_SubmitForm" name="unValidedText" time="0.027">
    <failure type="java.lang.AssertionError" message="expected [Thanks for submitting the formA] but found [Thanks for submitting the form]">
      <![CDATA[java.lang.AssertionError: expected [Thanks for submitting the formA] but found [Thanks for submitting the form]
at com.vndly.testCases.TC_SubmitForm.unValidedText(TC_SubmitForm.java:74)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- unValidedText -->
</testsuite> <!-- Test -->
